main/include/WeaveProjectConfig.h

```diff
@@ -159,4 +159,11 @@
 #define WEAVE_CONFIG_EVENT_LOGGING_DEFAULT_IMPORTANCE nl::Weave::Profiles::DataManagement::Debug
 #endif // BUILD_RELEASE
 
+/**
+ * WEAVE_DEVICE_CONFIG_LOG_MESSAGE_MAX_SIZE
+ *
+ * The maximum size (in bytes) of a debug logging message.
+ */
+#define WEAVE_DEVICE_CONFIG_LOG_MESSAGE_MAX_SIZE 512
```

Comment:
I'm wondering why we have log messages that are longer than 256 bytes.  That seems unnecessary.  Perhaps we should break up those messages, rather than increasing memory usage?

Reply:
Understood. Here is the PR:
https://github.com/openweave/openweave-core/pull/533

Closing this one.